src/main/java/utils/GcpClient.java

```diff
@@ -0,0 +1,55 @@
+package utils;
+
+import com.google.cloud.bigquery.*;
+import java.io.IOException;
+
+public class GcpClient {
+
+    public static int countBqQuery(String table) throws IOException {
+
+        String projectId = SeleniumHelper.readParameters("Project-ID");
+        String datasetName = SeleniumHelper.readParameters("Data-Set");
+        String tableName = SeleniumHelper.readParameters(table);
+        String query = "SELECT count(*) "
+                + " FROM `"
+                + projectId
+                + "."
+                + datasetName
+                + "."
+                + tableName
+                + "`";
+        return query(query);
+    }
+
+    public static void dropBqQuery(String table) throws IOException {
+
+        String projectId = SeleniumHelper.readParameters("Project-ID");
+        String datasetName = SeleniumHelper.readParameters("Data-Set");
+        ;
+        String tableName = SeleniumHelper.readParameters(table);
+        String query = "DROP TABLE `"
+                + projectId
+                + "."
+                + datasetName
+                + "."
+                + tableName
+                + "`";
+
```

Comment:
Do you want to run the query?

Reply:
yes, i want to get the count of records transferred

Reply:
So I don't see where you run the query. I see only where you make the query string

Reply:
just check the return statement it is calling the query function executing the string

Reply:
There is no return statement in this method